config/config.go

```diff
@@ -13,11 +13,18 @@ const DefaultNamespace = "default"
 const ResultsDatastreamIndexPrefix = "logs-cis_kubernetes_benchmark.findings"
 const MetadataDatastreamIndexPrefix = ".logs-cis_kubernetes_benchmark.metadata"
 
+type ProcessInputConfiguration struct {
```

Comment:
I thought about it a lot - Should the process configuration should be a part of the config or part of the process struct. I found reason in both.

In the end, I decided to put it here since I believe that the config is more "external" configuration than "internal".

I mean, the configuration of the process can be defined out of the general config and not the other way around.